ui/raidboss/data/04-sb/dungeon/temple_of_the_fist.js

```diff
@@ -3,7 +3,78 @@
 // The Temple of the Fist
 [{
   zoneRegex: /^The Temple Of The Fist$/,
+  timelineFile: 'temple_of_the_fist.txt',
+  timelineTriggers: [
+    {
+      id: 'Temple Pounce',
+      regex: /Pounce/,
+      beforeSeconds: 5,
+      condition: function(data) {
+        return data.role == 'healer' || data.role == 'tank';
+      },
+      infoText: {
+        en: 'Tank buster',
+      },
+    },
+    {
+      id: 'Temple Cardinal Shift',
+      regex: /Cardinal Shift/,
+      beforeSeconds: 5,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      infoText: {
+        en: 'AoE',
+      },
+    },
+  ],
   triggers: [
+    {
+      id: 'Temple Electric Burst',
+      regex: / 14:1FD6:Coeurl (?:Sruti|Smriti) starts using Electric Burst/,
```

Comment:
For the purposes of making translations easier, can you make these two separate triggers?

Reply:
Sure. Is it possible that at some point we will get to a place where we can do direct translation replacements within the regex, such that we could do something like this?

`14:1FD6:(?:Coeurl Sruti|Coeurl Smriti) starts using Electric Burst`

I'll change it for now, of course, but it seems a shame to need two triggers for an identical mechanic.

Reply:
No, that's fair.  I think your proposed replacement will work with the translation scripts.  It's really about keeping the mob name together.

Reply:
Okay. I'll keep that in mind for future work.